README.md

```diff
@@ -35,7 +35,7 @@ with the following command:
 $ go install github.com/arsham/blush@latest
 ```
 
-Make sure you have `go>=1.18` installed.
+Make sure you have `go>=1.21` installed.
```

Comment:
I'm always up for using new versions. But I think this would restrict people who haven't, don't want to, or can't upgrade Go's version. I would keep this at the minimum required version.